common.js

```diff
@@ -1943,6 +1943,233 @@ const allSoloCars = {
       '2012': ['gs']
     },
   },
+  'Subaru': {
+    'BRZ (non-Performance Package)': {
+      '2013': ['ds', 'stx', 'csp', 'dsp', 'ssc'],
+      '2014': ['ds', 'stx', 'csp', 'dsp', 'ssc'],
+      '2015': ['ds', 'ssc', 'stx', 'dsp'],
+      '2016': ['ds', 'ssc', 'stx', 'dsp'],
+      '2017': ['ds', 'stx', 'dsp'],
+      '2018': ['ds', 'cs', 'stx', 'dsp'],
```

Comment:
This should be the same as 2017. You cannot be eligible for both `ds` and `cs`. This is `ds` since it is the non-performance package.